kubernetes/schema_pod_spec.go

```diff
@@ -524,7 +525,7 @@ func volumeSchema(isUpdatable bool) *schema.Resource {
 					Type:         schema.TypeString,
 					Description:  "Target directory name. Must not contain or start with '..'. If '.' is supplied, the volume directory will be the git repository. Otherwise, if specified, the volume will contain the git repository in the subdirectory with the given name.",
 					Optional:     true,
-					ValidateFunc: validateAttributeValueDoesNotContain(".."),
+					ValidateFunc: validation.StringNotInSlice([]string{".."}, true),
```

Comment:
This is not quite the same as `validateAttributeValueDoesNotContain`. 

`StringNotInSlice` will only validate that the whole value is not in the given slice, whereas `validateAttributeValueDoesNotContain` is checking that the value does not _contain_ the given string. 

This will fail for `..` but pass on the value `../test` for example.

Reply:
Thanks, I'll check this case and see if can get it fixed.

Reply:
Done, and I added a test case for it.